Rocket.Chat/Managers/Uploader/UploadManager.swift

```diff
@@ -12,11 +12,18 @@ import SwiftyJSON
 public typealias UploadProgressBlock = (Int) -> Void
 public typealias UploadCompletionBlock = (SocketResponse?, Bool) -> Void
 
-struct FileUpload {
+class FileUpload {
```

Comment:
Why? 🤔

Reply:
@cardoso to be able to alter the filename (structs are constants once initialised), otherwise I have to alter the behaviour of the UploadManager (filename as separate parameter or other param than FileUpload) or create a new object with the same data as the old FileUpload but the changed name.

Reply:
@cardoso So should I change the uploadTo* funcs and add an additional name: String?

Reply:
@soundstorm I guess the uploadTo makes sense instead of changing the from `struct` to a `class`.